patches/tModLoader/Terraria/ModLoader/VanillaDamageClasses.cs

```diff
@@ -56,6 +56,10 @@ public class SummonDamageClass : VanillaDamageClass
 		protected override string LangKey => "LegacyTooltip.53";
 
 		protected override float GetBenefitFrom(DamageClass damageClass) => damageClass == Generic ? 1f : 0f;
+
+		public override bool AllowStandardCrits() => false;
```

Comment:
This line looks odd at a glance.
Mind adding a comment explaining in to the code?
I would have expected this to appear in SummonDamageClass instead.

Reply:
it is in SummonDamageClass though

Reply:
Oh, my bad, missed the line skip bar on the diff.
Looks good, apologies for confusion.

(Please mark as resolved, I for some reason can't despite being the reviewer.)